### ISSUE TITLE
doctl .deb package

### ISSUE THREAD

**Issue by maddsua** [digitalocean/doctl#1624]
**What is the problem this feature would solve? Please describe.**

The current installation process involves multiple manual actions and completely avoids any sort of package management.

**Describe the solution you'd like**

Create a .deb release artifact would allow it to be installed with a single command, just as remove/update the package more easily

**Additional context**

For example, railway's CLI is released as a deb package and installing that is way more convenient than fiddling with the binary directly

**Comment by Faizan-Alam-1** [comment]
@maddsua Yes, it's a great suggestion. I would love to work on this if the maintainers of the project allow it. @bentranter
